scipy/stats/_stats_py.py

```diff
@@ -9531,3 +9533,94 @@ def rankdata(a, method='average', *, axis=None, nan_policy='propagate'):
         result[nan_indexes] = np.nan
 
     return result
+
+
+def expectile(a, alpha=0.5, *, dtype=None, weights=None):
+    r"""Compute the expectile.
+
+    Expectiles are a generalization of the expectation in the same way as
+    quantiles are a generalization of the median. The expectile at level
+    `alpha = 0.5` is the mean (average). See Notes for more details.
+
+    Parameters
+    ----------
+    a : array_like
+        Input array or object that can be converted to an array.
+    alpha : float
```

Comment:
```suggestion
    alpha : float, default: 0.5
```

Reply:
`default: 0.5` or `default=0.5`?

Reply:
According to the numpydoc style guide:
![image](https://user-images.githubusercontent.com/6570539/198907694-9d8c28d9-0a26-4e02-9e6c-7b2b5c66c7b2.png)

and SciPy does not have a consistent standard.